### PR TITLE
Add desktop version with media query

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Achievements.

- Add a desktop version of the landing page
- Add a desktop version of the project gallery.
- Add a desktop version About me.
- Add a desktop version of the contact form.
:alien:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,6 +81,7 @@ <h3 class="profession">I'am a Software Developer</h3>
     <div class="button1"><button id="button1" type="button">See Project</button>
     </div>
 </section>
+<div class="projects">
```
Comment (topeogunleye) [inline index.html:84]: - [ ] [REQUIRED] Kindly ensure that the `See Project` button is only visible when a project is hovered on with a mouse and also the dark opacity should disappear when a project is hovered on with a mouse as seen below

Screenshot from 2022-03-24 22-18-05

File: style.css [hunk 2]
```diff
@@ -411,6 +492,98 @@ ul {
...
+  }
+
+  .line-card1 {
+    width: 100%;
+  }
+
+  .img1 {
+    padding-top: 0;
+    padding-right: 0;
+    width: 70%;
```
Comment (topeogunleye) [inline style.css:578]: - [ ] [REQUIRED] Kindly increase the width of your image to take up to 60% of the page's width. The *Featured project* image should take 60% of the page's width as seen below, and the description takes the remaining 40%

| Figma design | Your design |
| --- | --- |
| Screenshot from 2022-03-24 22-35-52 | Screenshot from 2022-03-24 22-36-52 |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Project Review
> # Changes Required 
> ### Hi @Artbsc1992,
> Nice job, you are almost there.
> 
> 

> 
> ## (Highlights) Good Points: 
> * Great interpretation of the figma design. :heavy_check_mark:
> * Github flow is used. :heavy_check_mark: 
> 
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my replies in the inline comments.
> 
> ## [Optional] suggestions:
>
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Artbsc1992) [reply index.html:84]: @topeogunleye Hello, thank you for the feedback, two thoughts:

- I know the pictures do not interact In the same way I thought that would be doing it by using Js. 
- The template is not giving me the background image to those cards, so I guess I need to use some random Image?.

Thank you in advance for your response.

**Review by smunozmo — APPROVED** [review]
Hi @Artbsc1992 , 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (topeogunleye) [reply index.html:84]: Replied through slack

**Pull request merged**